core/tauri-macros/src/lib.rs

```diff
@@ -151,3 +154,35 @@ pub fn do_menu_item(input: TokenStream) -> TokenStream {
   let tokens = parse_macro_input!(input as menu::DoMenuItemInput);
   menu::do_menu_item(tokens).into()
 }
+
+/// Convert a .png or .ico icon to an Image
+/// for things like `tauri::tray::TrayIconBuilder` to consume,
+/// relative paths are resolved from `CARGO_MANIFEST_DIR`, not current file
+///
+/// ### Examples
+///
+/// ```ignore
+/// TrayIconBuilder::new().icon(icon_image!("./icons/32x32.png")).build().unwrap();
+/// ```
+///
+/// Note: this stores the image in raw pixels to the final binary,
+/// so keep the icon size (width and height) small
+/// or else it's going to bloat your final executable
+#[proc_macro]
+pub fn icon_image(tokens: TokenStream) -> TokenStream {
+  let path = PathBuf::from(parse2::<LitStr>(tokens.into()).unwrap().value());
```

Comment:
maybe instead of unwrapping we do a compile error here?

Reply:
Yeah, I'm just getting started on those macros, need to learn a bit more